auth-web/src/components/auth/manage-business/BusinessLookup.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <!-- once in Summary state, need to re-mount to reuse this component -->
+  <div id="business-lookup" v-if="state !== States.SUMMARY">
+    <v-autocomplete
+      :hide-no-data="state != States.NO_RESULTS"
+      :items="searchResults"
+      :loading="state == States.SEARCHING"
+      :name="Math.random()"
+      :search-input.sync="searchField"
+      append-icon="mdi-magnify"
+      autocomplete="chrome-off"
+      autofocus
+      class="mt-5 mb-n2"
+      filled
+      hint="For example: &quot;Joe's Plumbing Inc.&quot;, &quot;BC1234567&quot;, &quot;FM1234567&quot;"
```

Comment:
This is how you get double quotes in a string in the template (without resorting to binding a variable in the script).